boards/arm/nucleo_f446ze/doc/index.rst

```diff
@@ -0,0 +1,258 @@
+| NVIC        | on-chip    | nested vector interrupt controller  |
++-------------+------------+-------------------------------------+
+| UART        | on-chip    | serial port                         |
++-------------+------------+-------------------------------------+
+| PINMUX      | on-chip    | pinmux                              |
++-------------+------------+-------------------------------------+
+| GPIO        | on-chip    | gpio                                |
++-------------+------------+-------------------------------------+
+| PWM         | on-chip    | pwm                                 |
++-------------+------------+-------------------------------------+
+| I2C         | on-chip    | i2c                                 |
++-------------+------------+-------------------------------------+
+| USB         | on-chip    | usb                                 |
++-------------+------------+-------------------------------------+
+| Backup SRAM | on-chip    | Backup SRAM                         |
++-------------+------------+-------------------------------------+
+| CAN 1/2     | on-chip    | Controller Area Network             |
++-------------+------------+-------------------------------------+
+| ADC         | on-chip    | Analog Input                        |
++-------------+------------+-------------------------------------+
+| DAC         | on-chip    | Analog Output                       |
++-------------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f446ze/nucleo_f446ze_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo F446ZE Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+Available pins:
+---------------
+.. image:: img/nucleo_f446ze_zio_left_2019_8_29.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Zio/Arduino connectors (left)
+.. image:: img/nucleo_f446ze_zio_right_2019_8_29.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Zio/Arduino connectors (right)
+.. image:: img/nucleo_f446ze_morpho_left_2019_8_29.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Morpho connectors (left)
+.. image:: img/nucleo_f446ze_morpho_right_2019_8_29.png
+   :width: 720px
+   :align: center
+   :height: 540px
+   :alt: Nucleo F446ZE Morpho connectors (right)
+
+For mode details please refer to `STM32 Nucleo-64 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_2_TX : PD5
+- UART_2_RX : PD6
+- UART_3_TX : PD8
+- UART_3_RX : PD9
+- USER_PB   : PC13
+- LD0       : PB0
+- LD1       : PB7
+- LD2       : PB14
+- I2C1_SDA  : PB9
+- I2C1_SCL  : PB8
+- I2C2_SDA  : PF0
+- I2C2_SCL  : PF1
+- SPI1_NSS  : PA4
+- SPI1_SCK  : PB3
+- SPI1_MISO : PB4
+- SPI1_MOSI : PB5
```

Comment:
Pin assignment should be changed according to changes made in DeviceTree
At least SPI1 (but could be good to check others)